README.md

```diff
@@ -54,7 +54,7 @@ Install with `pip`:
 
 `❯ pip install huak --pre`
 
-Install with `cargo`:
+Install with `cargo` (minimum rust version 1.62.0):
```

Comment:
I'd like to move towards a README that will need less and less maintenance over time. I think adding `rust-version` to the Cargo.toml should be sufficient to imply this. So maybe we remove this.